Source/WebCore/Modules/applicationmanifest/ApplicationManifestParser.h

```diff
@@ -46,6 +47,7 @@ class ApplicationManifestParser {
 
     URL parseStartURL(const JSON::Object&, const URL&);
     ApplicationManifest::Display parseDisplay(const JSON::Object&);
+    std::optional<WebCore::ScreenOrientationLockType> parseOrientation(const JSON::Object&);
```

Comment:
We're in WebCore so no `WebCore::`

Reply:
Can this function be marked as `const` or is there something that present it?

Reply:
Yes, I think it could. 

I think the other parser method s too could be `const`. I'll do the others as a followup.